dbfit-java/derby/src/integration-test/java/dbfit/environment/DerbyRegressionSupportStoredProcs.java

```diff
@@ -0,0 +1,31 @@
+package dbfit.environment;
+
+import java.sql.SQLException;
+import java.sql.DriverManager;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+public class DerbyRegressionSupportStoredProcs {
+
+    public static void MakeUser() throws SQLException {
+        Connection conn = null;
+        PreparedStatement stmt = null;
+        ResultSet res = null;
+
+        try {
+            // "jdbc:default:connection" tells the DriverManager to use the existing connection.
+            conn = DriverManager.getConnection("jdbc:default:connection");
+            String sql = "INSERT INTO Users (Name, UserName) VALUES ('user1', 'fromproc')";
+            stmt = conn.prepareStatement(sql);
```

Comment:
Can we use the [try-with-resources](https://github.com/dbfit/dbfit/blob/3b58e7255d78b81edcf3779bc9671ef2593c9e78/dbfit-java/netezza/src/main/java/dbfit/environment/NetezzaEnvironment.java#L69) syntax here (both for the statement and the connection)?